pkg/policyreport/policyreport.go

```diff
@@ -104,6 +104,11 @@ func updateResults(oldReport, newReport map[string]interface{}, aggregatedReques
 		return nil, hasDuplicate, err
 	}
 
+	if summaryResults == nil {
```

Comment:
It shouldn't be possible for this to ever be nil. We set it just above. The `mapToStruct` wouldn't result in this being nil in any scenario I can come up with.

Reply:
Ignore that, I'm mistaken.

Reply:
No, actually I was looking at the wrong version. Your fix was pushed to 1.5.2 and the issue is with 1.5.1. 

Sorry for the noise. Closing the PR.